src/arco_era5/update_co.py

```diff
@@ -0,0 +1,125 @@
+import apache_beam as beam
+import logging
+import subprocess
+import tempfile
+import os
+import datetime
+import xarray as xr
+import zarr as zr
```

Comment:
Let's keep the zarr import  name consistent (just zarr).

Reply:
Ok Updated.